docs: adding Breeze Agent for third party agent page

andrestorres123 opened the pull request:
Adding open source web researcher agent to the third party page

eyurtsev commented:
Hi @andrestorres123 what's the repo url on github? https://github.com/andrestorres123/breeze-agent?

andrestorres123 commented:
Hey @eyurtsev ! yes that's correct!

https://github.com/andrestorres123/breeze-agent

eyurtsev commented:
@andrestorres123 could you update pyproject.toml (author) and license in your package?

andrestorres123 commented:
Hi @eyurtsev ! Apologies for the oversight!

- I updated the pyproject.toml and the license with the correct information. 

- I made the package match the name I was submitting the PR with (breeze-agent) https://pypi.org/project/breeze-agent/

- I also added a USAGE section on the README file.

Let me know if these work or if there's anything else I need to update. Appreciate the time you took to provide feedback!

eyurtsev commented on docs/_scripts/third_party_page/packages.yml:
```diff
@@ -3,6 +3,9 @@ packages:
   - name: "trustcall"
     repo: "hinthornw/trustcall"
     description: "Tenacious tool calling built on LangGraph"
+  - name: "breeze"
```
```suggestion
  - name: "breeze-agent"
```
